docs/content/concepts/layer/control/catalog/_index.md

```diff
@@ -85,157 +88,271 @@ See [examples](#content-examples) of OSCAL catalogs.
+
+So for example, in its back matter the catalog XML for SP 800-53 may have something like:
+
+{{<highlight xml "nowrap=false" >}}
+<resource uuid="27847491-5ce1-4f6a-a1e4-9e483782f0ef">
+  <title>OMB A-130</title>
+  <citation>
+    <text>Office of Management and Budget Memorandum Circular A-130,
+    <em>Managing Information as a Strategic Resource</em> , July 2016.</text>
+  </citation>
+</resource>
+{{< / highlight >}}
+
+(*Note*: in the actual data, this resource is also provided with a link, not shown here.)
+
+The `uuid` value on this `<resource>` serves as its ID for internal links targeting this element. Incidentally, a survey of the catalog source data in this case also shows this particular reference is made from 96 different controls.
+
+Note that here, the code phrase used in the source to identify the bibliographic item (such as "OMB A-130") is regarded as a presentational artifact, not as a control point or formal identifier. This allows both freedom and fidelity to anomalous (and potentially conflicting) data sources, while providing for the linkage itself to be robust, expressed by means of a UUID. In rendition or display, any link to this resource can retrieve and show a title, citation text or outbound link as needed, where they are available.
+
+#### Parts
+
+After the links come the constituent pieces or `parts` of the control, as `<part>` elements (in JSON or YAML, members of a `parts` array), representing the main text, subordinate text and any subordinate parts or associated information (aside from control enhancements, which are considered controls in themselves, as described below).
+
+Like properties (`<prop>` elements) parts can be distinguished for semantic purposes by their names. Sets of names for known part types can be formalized at the level of either OSCAL or its application(s), or left informal for consumers and interchange partners to detect and respect as appropriate. Like parameters and controls, parts may also have properties and links of their own. In this data, `<prop>` elements are also used at the part level (that is, directly within `<part>`) to capture information sometimes relevant (to the part's rendition, to document and establish fidelity to upstream sources where it appears, or for other reasons). This can also be seen in other examples not shown here, such as the parts given in OSCAL SP 800-53 to describe control assessment objectives, methods and processes.
+
+`statement` is the name assigned to parts that capture the normative text of the control itself. In SP 800-53, some statement parts (control text) appear articulated into grouped and enumerated line items (such as here in AC-1); in others, a simple paragraph or paragraph sequence is given.
```

Comment:
Minor, non-blocking: did we formalize the notion that part and statement are similarly organizational containers? I had never thought of it that way, and it kind of makes sense to me. I could be wrong, but personally feel it could cause confusion with our downstream practitioners and devs in the community.

Reply:
Discussed this when we met prior, this is now obsolete.